DashAI/front/src/components/experiments/RunnerDialog.jsx

```diff
@@ -92,8 +83,19 @@ function RunnerDialog({ experiment, expRunning, setExpRunning }) {
       } else if (error.request) {
         console.error("Request error", error.request);
       } else {
-        console.error("Unkown Error", error.message);
+        console.error("Unknown Error", error.message);
       }
+    } finally {
+      enqueueSnackbar(`${experiment.name} has finished running`, {
+        variant: "info",
+        anchorOrigin: {
+          vertical: "top",
+          horizontal: "right",
+        },
```

Comment:
According to snackbars pr, this is no longer necessary.

Reply:
This was fixed by merging with staging.